torch/fx/experimental/symbolic_shapes.py

```diff
@@ -192,7 +192,7 @@ def has_free_symbols(val: Union[SymInt, torch.Tensor]) -> bool:
 # Like free_symbols, but filtered to only report unbacked symbols
 def free_unbacked_symbols(x):
     # NB: keep synced with is_unbacked_symint
-    return {s for s in free_symbols(x) if s.name.startswith("i")}
+    return {s for s in free_symbols(x) if s.name.startswith("i") or s.name.startswith("f")}
```

Comment:
```suggestion
    return {s for s in free_symbols(x) if s.name.startswith(("i", "f"))}
```
More succinct and efficient.